deployment/ansible/group_vars/production.example

```diff
@@ -0,0 +1,241 @@
+---
+## GLOBAL APP SETTINGS
+# Which version of the app to install. Specifically, this designates a tag on quay.io that should be
+# used when pulling docker images. A list of valid tags can be found here:
+# https://quay.io/repository/azavea/driver-app?tag=latest&tab=tags
+# Generally, when setting up a new instance, you should use the highest-number tag available. The
+# 'latest' tag should be avoided when setting up new instances for production usage, because it may
```

Comment:
Should this include a link to the sem ver spec? I'm imagining someone upgrading from 1.1.1 to 1.3.4 and arguing they weren't told they shouldn't do it. Something like "Upgrades where only the last version number has changed should be seamless. Other upgrades may take more effort. For more information, see the semantic versioning spec at https://semver.org/spec/v2.0.0.html"

Reply:
Yeah, this is a good point. I decided to add some additional suggestions about consulting the changelog and setting up a testing instance.